apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -37,8 +42,12 @@ export const TestWorkflowLogsSidebar = ({ transactionId, workflow }: TestWorkflo
       refetchInterval: shouldRefetch ? 1000 : false,
     }
   );
-  const activity = activities?.[0];
-  const activityId: string | undefined = parentActivityId ?? activity?._id;
+
+  const activityId: string | undefined = parentActivityId ?? activities?.[0]?._id;
+  const { activity: latestActivity, isPending: isActivityPending, error: activityError } = usePullActivity(activityId);
+  const activity = latestActivity ?? activities?.[0];
```

Comment:
What is the use case of showing the first activity from the list of activities? If the activityId is present in the URL this component should load that activity. If it's not, the component doesn't render at all, as the activity table is full-width. Am I missing something?

Reply:
this is the Trigger Workflow - activity panel component ;)